language/move-lang/tests/move_check/translated_ir_tests/move/signer/move_to_missing_signer.exp

```diff
@@ -8,6 +17,15 @@ error[E04007]: incompatible types
   │         Invalid call of 'move_to'. Invalid argument for parameter '0'
   │         Expected: '&signer'
 
+error[E04016]: too few arguments
+   ┌─ tests/move_check/translated_ir_tests/move/signer/move_to_missing_signer.move:14:14
+   │
+14 │         () = move_to<R<bool>>(R<bool> { f: false });
+   │              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Don't need to fix this in this PR unless you want to, but I'm wondering if it would be worthwhile to have this "Found YY arguments(s) here" point to the parameters of the function and not the whole function call?

Reply:
Yeah I don't have a loc for just the args I don't think, so another PR